web/js/index.js

```diff
@@ -22,7 +22,26 @@ const loadTexture = (path, loader, onProgress) => {
 }
 
 // Create scene and camera
-let scene = new THREE.Scene();
+var scene;
+var scene1 = new THREE.Scene();
+var scene2 = new THREE.Scene();
```

Comment:
Calling these _scene1_ and _scene2_ could get confusing if someone doesn't remember which is 1 or 2. We should update these to more descriptive names, like _moonScene_ and _spaceScene_, or something like that :smiley:

Reply:
Updated the names in the file.